src/ring_theory/localization.lean

```diff
@@ -1201,6 +1201,17 @@ end
 end localization
 end at_prime
 
+/-- If `R` is a field, then localizing at a submonoid not containing `0` adds no new elements -/
```

Comment:
Here and elsewhere -- if a docstring is a complete sentence, it should end in a period.
```suggestion
/-- If `R` is a field, then localizing at a submonoid not containing `0` adds no new elements. -/
```